contribs/gmf/src/editing/editFeatureComponent.js

```diff
@@ -422,6 +419,16 @@ function Controller(
    */
   this.features = null;
 
+  /**
+   * @type {?import("ol/layer/Vector.js").default}
+   */
+  this.highlightVectorLayer = null;
```

Comment:
Sorry, I don't have seen that before: It should be private no ? (written with a final underscore: `highlightVectorLayer_`)